tests/Tests/500__jupyterhub/test-folder-permissions.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation       Test Suite to verify installed library versions
+
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             JupyterLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
+Verify Folder Permissions
+    [Documentation]    It checks Access, Gid, Uid of /opt/app-root/lib and /opt/app-root/shares
+    [Tags]    ODS-486
+    ...       Tier2
+    @{list_of_images} =    Create List
+    Append To List
+    ...    ${list_of_images}
+    ...    s2i-minimal-notebook
+    ...    s2i-generic-data-science-notebook
+    ...    pytorch
+    ...    tensorflow
+    ${permissions} =    Create List    0775    0    1001
+    ${path} =    Create List    lib    share
+    FOR    ${img}    IN    @{list_of_images}
+        Verify The Permissions Of Folder In Image    ${img}    ${path}    ${permissions}
+    END
```

Comment:
could you re-format the test case to be more "high-level" as descripted in [style guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.4a2fb94fltk2) ?

Reply:
Read it and made changes accordingly